doc/glofas_rf.rst

```diff
@@ -0,0 +1,449 @@
+
+------------
+Preparations
+------------
+
+We need to prepare three things: The flood hazard maps, the extreme value distributions, and access to the CDS API.
+
+Copernicus Data Store API Access
+
+1. Register at the `Copernicus Data Store (CDS) <https://cds.climate.copernicus.eu/#!/home>`_ and log in.
+2. Check out the `CDS API HowTo <https://cds.climate.copernicus.eu/api-how-to>`_.
+   In the section "Install the CDS API key", copy the content of the black box on the right.
+3. Create a file called ``.cdsapirc`` in your home directory and paste the contents of the black box into it.
+
+  If you are unsure where to put the file and you are working on a Linux or macOS system, open a terminal and execute
+
+  .. code-block:: shell
+
+    cd $HOME
+    touch .cdsapirc
+
+  Now the file is created and can be opened with your favorite text editor.
+
+Create Data Directory
+^^^^^^^^^^^^^^^^^^^^^
+
+1. Enter the CLIMADA data directory.
+   If you did not change the default configuration, this is located in ``$HOME/climada/data``.
+
+  If this directory does not exist, make sure you activated the CLIMADA environment and execute
+
+  .. code-block:: shell
+
+    python -c "import climada"
+
+2. Create a new directory called ``glofas-computation`` in the data directory.
+
+Download Prepared Datasets from the ETH Research Collection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The data required for flood footprint computation has been uploaded to the `ETH Research Collection <https://www.research-collection.ethz.ch/>`_ for your convenience:
+
+* `Gumbel distribution fit parameters for historical GloFAS river discharge data <https://doi.org/10.3929/ethz-b-000587207>`_
+* `Flood hazard map of the World <https://doi.org/10.3929/ethz-b-000587202>`_
+
+Place the downloaded files in the ``glofas-computation`` data directory you created in the last section.
+If you did that, you can directly proceed with :ref:`computing flood footprints <compute>`.
+
+If not, you will have to follow the next steps to compute the input data yourself:
+
+Optional: Create Input Data Using ``setup``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you cannot download the data, you can re-create it on your machine using the :py:func:`climada_petals.hazard.rf_glofas.setup` function.
+To do so, follow the optional instructions below.
+
+First, download the global flood hazard map source data.
+Freely accessible global flood hazard maps are available from the Joint Research Centre (JRC) Data Catalogue as GeoTIFF files: https://data.jrc.ec.europa.eu/collection/id-0054
+
+1. Download all "Flood hazard map of the World" from the JRC Data Catalogue.
+   Click on each respective entry, navigate to the "Data access" section and click "Download".
+2. Unzip the downloaded files.
+3. Copy or move the unzipped folders into the data directory.
```

Comment:
Correct. I think this part is now obsolete because I uploaded the prepared files to the ETH research collection and will provide helper functions that automatically download them (see my other comment)

Reply:
But you might want to have reproducible code to produce the pre-computed files in the ETH research collection. So, I would still suggest to implement this.

Reply:
These files are downloaded and merged now with `setup_all`